src/OrchardCore.Modules/OrchardCore.Users/Views/ExternalLoginSettings.Edit.cshtml

```diff
@@ -52,34 +65,34 @@
 
     // don't format here.
     const suggestion = `/**
-            * Use the loginProvider and externalClaims properties of the context variable to inspect
-            * who authenticated the user and with what claims. Check currentRoles property and apply
-            * your business logic to fill the rolesToAdd and rolesToRemove arrays in order to update
-            */
-            const context = {} as Context;
-            type Context = {
-                readonly user: {
-                    userName: string;
-                    userRoles: string[];
-                    userClaims: { claimType: string; claimValue: string; }[];
-                    userProperties: { [key: string]: string };
-                },
-                readonly loginProvider: string,
-                rolesToAdd: string[];
-                rolesToRemove: string[];
-                claimsToUpdate: { claimType: string; claimValue: string; }[];
-                claimsToRemove: { claimType: string; claimValue: string; }[];
-                propertiesToUpdate: { [key: string]: string };
-                externalClaims: readonly [{
-                    valueType: string;
-                    type: string;
-                    value: string;
-                    subject: string;
-                    issuer: string;
-                    originalIssuer: string;
-                    properties: { [key: string]: string };
-                }]
-            }`
+                                                            * Use the loginProvider and externalClaims properties of the context variable to inspect
```

Comment:
I guess the indentation is not deliberate:

![{1473DFF0-C1B2-4EF4-8EB7-A9C91A10A06C}](https://github.com/user-attachments/assets/d90a5f36-2490-4042-a2bd-297bf64181a8)

Reply:
yeah VS did some magic here. Fixed it.